app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str | None:
+    if len(command.split()) != 3:
+        return "Not enough arguments in command"
+    command, source_file_name, new_path = command.split()
```

Comment:
Don't use `command.split()` twice. You should create separate variable for storing the result of `command.split()` and use it where you need